recipes/chelsa/recipe.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+from pangeo_forge_recipes.patterns import ConcatDim, MergeDim, FilePattern
+from pangeo_forge_recipes.recipes import XarrayZarrRecipe
+
+months = np.arange(1,13) 
+variables = ['cmi',
+            'hurs',
+            'pet',
+            'pr',
+            'sfcWind',
+            'tas',
+            'tasmax',
+            'tasmin',
+            'tcc',
+            'vpd'
+           ]
+
+def make_url(variable, month):
+    '''
+    Takes two keys and translates it into a URL into a data file
+    '''
+    if variable == 'pet':
+        variable_filename = 'pet_penman'
+    else:
+        variable_filename = variable
+    return (
+        'https://os.zhdk.cloud.switch.ch/envicloud/chelsa/chelsa_V2/GLOBAL/'
+        f'climatologies/1981-2010/{variable}/CHELSA_{variable_filename}_{month:02}_1981-2010_V.2.1.tif'
```

Comment:
Will Xarray open tifs with no special arguments?

Reply:
Good call! It's unlikely, so we may need to specify `xr.open_dataset(..., engine="rasterio")`, but I'm unsure where in the recipe this specification is made. Do you have any suggestions?

Reply:
👋  hi Hillary! Great to have you here!

I would start by _downloading and opening a single file manually to figure out what xarray arguments are needed_.  Then you would need to update your recipe options. Some of the [customization arguments](https://pangeo-forge.readthedocs.io/en/latest/pangeo_forge_recipes/api_reference.html#pangeo_forge_recipes.recipes.XarrayZarrRecipe) are:
- `xarray_open_kwargs`, e.g. `xarray_open_kwargs={'engine': 'rasterio'}`
- `copy_input_to_local_file` needs to be set to `True` if rasterio requires a local file (as opposed to an fsspec file-like object)

Reply:
Thank you @rabernat! That's a great suggestion. I'll have a crack at it and modify the recipe accordingly.